Cargo.toml

```diff
@@ -43,7 +43,7 @@ once_cell = "1.16.0"
 pin-project-lite = "0.2.9"
 rand = "0.8.5"
 reqwest = { version = "0.12.4", default-features = false }
-ruma = { git = "https://github.com/ruma/ruma", rev = "75e8829bec0b7bc5332860e1fb2df658d5c71d66", features = [
+ruma = { git = "https://github.com/toger5/ruma/", rev = "d7094e68e0ed6875f97e0928101d90654d054078", features = [
```

Comment:
Let's wait on https://github.com/ruma/ruma/pull/1829 to be merged before going forward.